oracle/feed_binance.go

```diff
@@ -91,6 +90,10 @@ func (f *binancePriceFeed) OracleType() oracletypes.OracleType {
 	return oracletypes.OracleType_PriceFeed
 }
 
+func (f *binancePriceFeed) AssetPair() *oracletypes.AssetPair {
```

Comment:
why not change the current interface so the puller returns directly the final pricedata?

I did a PoC here, and some extreme clean up too
https://github.com/InjectiveLabs/injective-price-oracle/pull/17

Reply:
let me check 😉

Reply:
I merged it here 🙏🏻